cookbooks/fb_iptables/spec/fixtures/default/iptables_multi_chain

```diff
@@ -8,10 +8,10 @@
 COMMIT
 
 *filter
+-N LOG_DB
 :INPUT ACCEPT [0:0]
 :OUTPUT ACCEPT [0:0]
 :FORWARD ACCEPT [0:0]
-:LOG_DB  [0:0]
```

Comment:
You sure this isn't actually needed?

Reply:
It's actually a syntax error. It'll break it loading. You can, on some versions do `:LOG_DB - [0:0]`, but it's a no-op and never necessary. BUt it's a 3-column field we're generating 2 columns for.